spec/workflow/engine_spec.rb

```diff
@@ -398,6 +398,63 @@ module Kuroko2::Workflow
           expect(token.status_name).to eq 'finished'
         end
       end
+
+      context 'with retry' do
+        let!(:definition) do
+          create(:job_definition_with_instances, script: <<-EOF.strip_heredoc)
+            retry: count=3 sleep_time=1
+              noop:
+              noop:
+          EOF
+        end
+
+        let(:root){ ScriptParser.new(token.script).parse(validate: false) }
```

Comment:
[MUST] Put a whitespace before { of brace blocks.

https://github.com/cookpad/styleguide/blob/master/ruby.en.md#method-calls